centml/compiler/prediction_sample.py

```diff
@@ -0,0 +1,23 @@
+import time
```

Comment:
these types of files shouldn't be part of the main source code. Move it either to the test/ dir or to the scripts/ dir.

Reply:
It was going to be de deleted anyways, left it in there for initial context